enhancements/network-observability/network-observability.md

```diff
@@ -199,18 +308,18 @@ not be an option to selectively choose some OVS to be exporters only.  There
 will be a configuration parameter to set the sampling rate.  Other parameters
 such as the cache size may also be exposed.
 
-#### Flow Collector
-The Flow Collector is responsible for receiving and normalizing the data
-sent by the NetFlow exporter.  This component is [GoFlow 2](https://github.com/netsampler/goflow2).
-The enhancement is at [Enable flows collection](https://github.com/mariomac/enhancements/blob/enable-flows-collection/enhancements/network/enable-flows-collection.md).
+#### eBPF agent
+Extended Berkeley Packet Filter (eBPF) is used to safely and efficiently extend 
+the capabilities of the kernel without requiring to change kernel source code or 
+load kernel modules.  The agent will allows collecting and aggregating all the 
+ingress and egress flows on Linux hosts to send IPFIX data.
 
-#### Flow Enricher
-The Flow Enricher enhances the IPFIX data with Kubernetes-related
-information.  It watches for updates on pods, services and daemonsets
-using the Kubernetes API.  It merges data from Flow Collector and
-Kubernetes to be able to display pod-to-pod traffic.  It is also
-responsible for writing out the data to Storage.  See Network Observability
-Flow Enricher enhancement for more details (TBD).
+#### FLP
+Flow-Logs Pipeline (FLP) is an observability tool that consumes raw network 
+flow-logs in their original format (NetFlow v5,v9 or IPFIX) and uses a pipe-line 
+to enhances the IPFIX data with Kubernetes-related information, transform the logs
+into time series metrics in prometheus format and in parallel transform and persist
+the logs also into loki.
```

Comment:
I'd also mention that is able to ingest eBFP flows in binary format (protobuf+GRPC) or JSON flows directly from Kafka.

Reply:
Sure, should we also update FLP [Overview section](https://github.com/netobserv/flowlogs-pipeline#overview) ?

Reply:
Ah, yes!